src/main/java/at/hannibal2/skyhanni/features/chat/StashCompact.kt

```diff
@@ -63,12 +63,12 @@ object StashCompact {
 
     private val config get() = SkyHanniMod.feature.chat.filterType.stashMessages
 
-    private var lastMaterialCount = 0
-    private var lastDifferingMaterialsCount = 0
-    private var lastType = ""
+    private var lastMaterialCount = -1
```

Comment:
please use nullable instead of -1

Reply:
Ig
I didn't because the reason for changing it is just if it somehow doesn't get changed before sending the compact message (because that did happen before the fix), it'll show something more clear that it's an error
But null isn't actually used